create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/PublicationConverter.java

```diff
@@ -0,0 +1,4 @@
+package no.sikt.nva.scopus;
+
```

Comment:
Jeg forstår at du be oss om å bruke TDD utvikling så lenge vi jobber inn i NVA koden?

Reply:
It is just a suggestion. From a reviewer's point of view I do not feel comfortable approving incomplete code that I do not know what it does. And I am not sure what this class is supposed to do and what is its purpose. I see that ScopusHandler depends on this class. 

So here I start to imagine things, probably they want to do this or that, so that is why they have added this class. But what the reviewer is imaging, is not what the coder imagined. So the reviewer approves code that has never existed and probably will never exist. 

In short, I cannot approve the existence of this class, since I do not know its purpose.